resources/i18n/de/strings.xml

```diff
@@ -170,17 +170,82 @@
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Subtitle - SH -->
+  <string name="MASK_STATE_MASK_REQUIRED_SUBTITLE_TEXT_SH">"Schleswig-Holstein\nSie sind nicht von der Maskenpflicht ausgenommen"</string>
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Subtitle - TH -->
+  <string name="MASK_STATE_MASK_REQUIRED_SUBTITLE_TEXT_TH">"Thüringen\nSie sind nicht von der Maskenpflicht ausgenommen"</string>
+
   <!-- XTXT: Mask State - MASK_REQUIRED Card Body -->
   <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT">"Von der Maskenpflicht sind alle Personen befreit, die innerhalb der letzten 3 Monate geimpft wurden oder genesen sind oder innerhalb der letzten 24 Stunden negativ getestet wurden."</string>
   <!-- XTXT: Mask State - MASK_REQUIRED Badge Text -->
   <string name="MASK_STATE_MASK_REQUIRED_BADGE_TEXT">"Maskenpflicht"</string>
 
   <!-- XHED: Mask State - MASK_OPTIONAL Card Title -->
   <string name="MASK_STATE_MASK_OPTIONAL_TITLE_TEXT">"Keine Maskenpflicht"</string>
-  <!-- XHED: Mask State - MASK_OPTIONAL Card Subtitle -->
-  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT">"Eine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - Federal -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_DE">"Regeln des Bundes\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - BW -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_BW">"Baden-Württemberg\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - BY -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_BY">"Bayern\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - BE -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_BE">"Berlin\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - BB -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_BB">"Brandenburg\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HB -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HB">"Bremen\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HH -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HH">"Hamburg\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - HE -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_HE">"Hessen\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - MV -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_MV">"Mecklenburg-Vorpommern\nEine Maske ist dennoch empfohlen"</string>
+  <!-- XTXT: Mask State - MASK_OPTIONAL Card Subtitle - NI -->
+  <string name="MASK_STATE_MASK_OPTIONAL_SUBTITLE_TEXT_NI">"Niedersachsen\nEine Maske ist dennoch empfohlen"</string>
```

Comment:
Why is the word "dennoch" in here? Is there a paragraph before which says something like "Von der Maskenpflicht ausgenommen"?

Reply:
This particular text sequence was dropped already 😅 You find the latest proposal in https://github.com/corona-warn-app/cwa-app-ccl/pull/106/files#diff-3526f651c4d797d161de635d89eff2562be6a1ed70ca0ad709556114c9c2164c. The _dennoch_ is still there, but it's embedded into a sentence.

Reply:
Fair enough, thanks!